ProgramParsing/Environment/MajorParser2021_2022.py

```diff
@@ -72,6 +75,52 @@ def _choose_x_course_list(self, i, info):
             i += 1
         return i, list
 
+    def _choose_option(self, i, info, program, relatedMajor, term):
+        option_num = 0
+        number_additional = 1
+        space_count = 0
+        while i < len(info):
+            line = info[i].strip().replace(" to ", "-")
+            # gets the option number
+            if line.split(" ")[0] == "Option":
+                option_num += 1
+                space_count = 0
+
+            # counts the empty lines
+            if line == "":
+                space_count += 1
+                if space_count == 5:
```

Comment:
wait so you should continue after the line=="" ? This is for clarity - I don't think it changes anything

Reply:
The counting of "" is used to determine, when all the options have been looked at. I think the number of blank lines corresponds to the bullet indentation found on the pages.